tests/functional/aws-node-sdk/test/bucket/testBucketVersioning.js

```diff
@@ -0,0 +1,131 @@
+import assert from 'assert';
+import { S3 } from 'aws-sdk';
+
+import getConfig from '../support/config';
+
+const bucket = `versioning-bucket-${Date.now()}`;
+
+describe('aws-node-sdk test bucket versioning', function testSuite() {
+    this.timeout(60000);
+    let s3;
+
+    // setup test
+    before(() => {
+        const config = getConfig('default', { signatureVersion: 'v4' });
+
+        s3 = new S3(config);
+    });
+
+    // createbucket test
+    it('should create a bucket', done => {
```

Comment:
probably makes more sense to put this in the before block rather than its own test.